lib/utils.js

```diff
@@ -7,7 +7,7 @@ exports.isSQLite = function isSQLite (sequelize) {
 exports.getImageMimeType = function getImageMimeType (imagePath) {
   var fileExtension = /[^.]+$/.exec(imagePath)
```

Comment:
I think this would be a better place to apply the ``.toLowerCase`` but the 1.6 code is not in development as much anymore so maybe this also works.

Reply:
I think it doesn't matter much. You would need to also move the `[0]` array access up, so the current version moves less code.